releases/production/pipeline.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
 spec:
   releaseName: pipeline
   chart:
-    git: git@github.com:biomage-ltd/pipeline
+    git: git@github.com:hms-dbmi-cellenics/pipeline
```

Comment:
Are these ones in `production/` (and the ones in `staging/` that are used by `default`) supposed to be changed?